### PR TITLE
Update Rule “review-videos-collaboratively/rule”

### PR DESCRIPTION
**Pull request by williamgreentreessw**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/review-videos-collaboratively/rule.md [hunk 1]
```diff
@@ -28,9 +27,12 @@ So you have just finished that sweet video edit and you are now ready & excited
 
 This is where collaborative viewing platforms like [Frame.io](https://frame.io) come in. 
 
+<iframe width="560" height="315" src="https://www.youtube.com/embed/rapbhpW3jPM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
```
Comment (tiagov8) [inline rules/review-videos-collaboratively/rule.md:30]: Hey @williamgreentreessw 

1. Please see https://ssw.com.au/rules/rule/ and follow the right markdown to add a video

File: rules/review-videos-collaboratively/rule.md [hunk 2]
```diff
@@ -115,6 +119,33 @@ Exporting feedback - to download all comments:
...
+| Advance Timecode with Frame Counter | ✅        | ❌         | ❌     | ❌           | ❌              | ❌         | ❌                   |
+| Unlisted Videos                     | ✅        | ✅         | ✅     | ✅           | ✅              | ❌         | ❌                   |
+| Versioning                          | ✅        | ✅         | ✅     | ❌           | ❌              | ❌         | ❌                   |
+| Timestamped comments                | ✅        | ❌         | ✅     | ✅           | ❌              | ❌         | ❌                   |
+| Project File Management             | ✅        | ✅         | ❌     | ❌           | ❌              | ❌         | ❌                   |
+| Downloadable                        | ✅        | ✅         | ✅     | Low Quality | Low Quality    | ❌         | ❌                   |
+| Draw on Video                       | ✅        | ❌         | ❌     | ❌           | ❌              | ❌         | ❌                   |
+| Export Comments                     | ✅        | ❌         | ❌     | ❌           | ❌              | ❌         | ❌<!--EndFragment--> |
+
+<br>
```
Comment (tiagov8) [inline rules/review-videos-collaboratively/rule.md:147]: Hey @williamgreentreessw 

1. Please remove this HTML from Markdown

File: rules/review-videos-collaboratively/rule.md [hunk 3]
```diff
@@ -28,9 +27,13 @@ So you have just finished that sweet video edit and you are now ready & excited
 
 This is where collaborative viewing platforms like [Frame.io](https://frame.io) come in. 
 
+'youtube: https://www.youtube.com/embed/rapbhpW3jPM'
+**Video What is Frame.io (2:16 min)**
```
Comment (tiagov8) [inline rules/review-videos-collaboratively/rule.md:31]: @williamgreentreessw 

1. Please follow https://ssw.com.au/rules/right-format-to-write-videos-time-length/ here

File: rules/review-videos-collaboratively/rule.md (hunk 2 above)
Comment (williamgreentreessw) [reply rules/review-videos-collaboratively/rule.md]: Removed HDML Tags

File: rules/review-videos-collaboratively/rule.md (hunk 1 above)
Comment (williamgreentreessw) [reply rules/review-videos-collaboratively/rule.md]: Done, followed rule to embed video

File: rules/review-videos-collaboratively/rule.md (hunk 3 above)
Comment (williamgreentreessw) [reply rules/review-videos-collaboratively/rule.md]: Fixed markdown

**Comment by williamgreentreessw** [comment]
As per our conversation I have made all changes.

**Pull request merged**